### ISSUE TITLE
YOLOv8 Supervision library dependency broken

### ISSUE THREAD

**Issue by cornking03** [roboflow/notebooks#64]
### Search before asking

- [X] I have searched the Roboflow Notebooks [issues](https://github.com/roboflow-ai/notebooks/issues) and found no similar bug report.

### Notebook name

[Track and Count Vehicles with YOLOv8 + ByteTRACK + Supervision](https://github.com/roboflow-ai/notebooks/blob/main/notebooks/how-to-track-and-count-vehicles-with-yolov8.ipynb)

### Bug

[Supervision ](https://github.com/roboflow/supervision) library was updated from 0.1.0 to 0.2.0. It looks like there was enough changed that this broke the imports.

![image](https://user-images.githubusercontent.com/84036755/217389807-56e1604c-b71d-47c5-9df4-6319256ad7d6.png)

### Environment

Google Colab

### Minimal Reproducible Example

See screenshot in `Bug` section

### Additional

I believe imports can be fixed using the following:

```py
from supervision.draw.color import ColorPalette
from supervision.geometry.core import Point
from supervision.video import VideoInfo, get_video_frames_generator, VideoSink
from supervision.notebook.utils import show_frame_in_notebook
from supervision.detection.core import Detections, BoxAnnotator
from supervision.detection.line_counter import LineZone, LineZoneAnnotator
```

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by SkalskiP** [comment]
Hi, @cornking03 👋🏻! Thanks a lot for the fast reaction 🙏🏻! The notebook should work now. I just pushed the fixed version. We released Supervision `0.2.0` and introduced many new cool features and breaking changes. For this notebook to work, you need to stay at `0.1.0`. 

Thank you for that issue once again!

**Comment by SkalskiP** [comment]
No updates. Looks like the problem is gone. I'm closing the issue.
